fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -866,6 +866,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int max_stream_load_batch_size_mb = 100;
 
+    /**
+     * Default stream load max txn num per BE
+     */
+    @ConfField(mutable = true)
+    public static int stream_load_max_txn_num_per_be = 5;
```

Comment:
Is it possible to default to unlimited so that the behavior is the same as before, otherwise there will be compatibility issues?

Reply:
Sure, already modified.